addon/components/_private/rdfa-editor/external-triple-editor.gts

```diff
@@ -0,0 +1,229 @@
+      >
+        <:title>Edit external triples</:title>
+        <:body>
+          <ExternalTripleForm
+            @onSubmit={{@onSubmit}}
+            id={{formId}}
+            @triple={{@triple}}
+          />
+        </:body>
+        <:footer>
+          <AuButtonGroup>
+            <AuButton form={{formId}} type='submit'>Save</AuButton>
+            <AuButton
+              @skin='secondary'
+              {{on 'click' @onCancel}}
+            >Cancel</AuButton>
+          </AuButtonGroup>
+        </:footer>
+      </AuModal>
+    </WithUniqueId>
+  </template>
+}
+interface ExternalTripleItemSig {
+  Args: {
+    trip: FullTriple;
+    index: number;
+    onRemove: (index: number) => void;
+    onEdit: (index: number) => void;
+  };
+}
+class ExternalTripleItem extends Component<ExternalTripleItemSig> {
+  get datatype() {
+    const trip = this.args.trip;
+    if (trip.object.termType === 'Literal') {
+      return trip.object.datatype.value;
+    }
+    return null;
+  }
+  get language() {
+    const trip = this.args.trip;
+    if (trip.object.termType === 'Literal') {
+      return trip.object.language;
+    }
+    return null;
+  }
+  <template>
+    <div class='au-u-padding-tiny'>
+      <p><strong>subject:</strong> {{@trip.subject.value}}</p>
+      <p><strong>predicate:</strong> {{@trip.predicate}}</p>
+      {{#if this.datatype}}
+        <p><strong>datatype:</strong> {{this.datatype}}</p>
+      {{/if}}
+      {{#if this.language}}
+        <p><strong>language:</strong> {{this.language}}</p>
+      {{/if}}
+      <p><strong>value:</strong> {{@trip.object.value}}</p>
+    </div>
+
+    <AuDropdown @icon={{ThreeDotsIcon}} role='menu' @alignment='left'>
+      <AuButton
+        @skin='link'
+        @icon={{PencilIcon}}
+        role='menuitem'
+        {{on 'click' (fn @onEdit @index)}}
+      >
+        Edit property
+      </AuButton>
+      <AuButton
+        @skin='link'
+        @icon={{BinIcon}}
+        role='menuitem'
+        class='au-c-button--alert'
+        {{on 'click' (fn @onRemove @index)}}
+      >
+        Remove property
+      </AuButton>
+    </AuDropdown>
+  </template>
+}
+
+interface Sig {
+  Args: { controller: SayController; node: ResolvedPNode };
+}
+export default class ExternalTripleEditor extends Component<Sig> {
+  @tracked
+  editModalOpen = false;
+  @tracked
+  indexBeingEdited?: number;
+  get node() {
+    return this.args.node;
+  }
+  get externalTriples(): FullTriple[] {
+    return this.node.value.attrs['externalTriples'] ?? [];
+  }
+  get controller() {
+    return this.args.controller;
+  }
+  get tripleBeingEdited(): Option<FullTriple> {
+    if (isSome(this.indexBeingEdited)) {
+      return this.externalTriples[this.indexBeingEdited];
+    }
+    return null;
+  }
+  closeModal = () => {
+    this.editModalOpen = false;
+  };
+  updateExternalTriples = (trip: FullTriple) => {
+    let tr;
+    if (isSome(this.indexBeingEdited)) {
+      console.log('tripe', trip);
```

Comment:
console.log

Reply:
fixed in [1c528ec](https://github.com/lblod/ember-rdfa-editor/pull/1229/commits/1c528ec9dc93bbeee04271788d37ed3c9dffeab1)